src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -129,23 +129,29 @@ const usePushNotificationHook = () => {
   };
 
   const __handleNotification = async (notification) => {
-    if (notification.data.type === 'feed' || notification.data.type === 'reaction') {
+    if (notification?.data?.type === 'topic' && notification?.data?.topic_name) {
+      navigation.navigate('TopicPageScreen', {
+        id: notification?.data?.topic_name
+      });
+    }
+
+    if (notification?.data?.type === 'feed' || notification?.data?.type === 'reaction') {
       navigation.navigate('PostDetailPage', {
         feedId: notification.data.feed_id,
         is_from_pn: true
       });
     }
-    if (notification.data.type === 'follow_user') {
+    if (notification?.data?.type === 'follow_user') {
       navigation.navigate('OtherProfile', {
         data: {
-          user_id: notification.data.user_id,
-          other_id: notification.data.user_id_follower,
-          username: notification.data.username_follower
+          user_id: notification?.data?.user_id,
+          other_id: notification?.data?.user_id_follower,
+          username: notification?.data?.username_follower
         }
       });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, it's important to ensure that `user_id`, `user_id_follower`, and `username_follower` exist before navigating to the 'OtherProfile' page. 
```diff
-    if (notification?.data?.type === 'follow_user') {
+    if (notification?.data?.type === 'follow_user' && notification?.data?.user_id && notification?.data?.user_id_follower && notification?.data?.username_follower) {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->